deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -324,7 +324,23 @@ spec:
     for this version. You can get it from the releases on GitHub.
 
         curl -Ls https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc3/kubefedctl-0.1.0-rc3-linux-amd64.tgz | tar xz
-
+    
+    ### Create a KubeFed custom resource
```

Comment:
The flow here is not very coherent. The previous section still talks about the install steps as if `the operators is kubefed`. I would like for us to list the three steps at the very top. Something like in-order to deploy kubefed, one needs to follow these two steps:
 - install the operator in a namespace/cluster scoped
 - create kubefed cr which is cluster/namespace scoped

Once the federation is running, a user can communicate with it using the kubefedctl
 - Talk about how one can get kubefedctl

Once you have downloaded kubefedctl, users can join the cluster using `kubefedctl join`.

That is the general workflow that doesn't make the user feel that they are going back and forth.